scss/navbar.scss

```diff
@@ -0,0 +1,18 @@
+header {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0px 5px 5px 0px $header-color;
+  display: flex;
+  font-size: $text-big;
```

Comment:
although it's not wrong to apply font sizes to containers It's preferred to target texts directly using a class to avoid assumpions